demo/public/app.js

```diff
@@ -47,7 +47,9 @@ var uiConfig = {
     firebase.auth.EmailAuthProvider.PROVIDER_ID
   ],
   // Terms of service url.
-  'tosUrl': 'https://www.google.com'
+  'tosUrl': 'https://www.google.com',
+  // Whether the display name should be displayed in Sign Up page.
+  'shouldDisplayName': false
```

Comment:
Open to changing the config property name for this and / or nesting it in `signInOptions`?